server/gitlab/api.go

```diff
@@ -11,6 +11,19 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+// GetProjectHooks gathers all the project level hooks from a single GitLab project.
+func (g *gitlab) GetProjectHooks(user *GitlabUserInfo, owner string, repo string) ([]*internGitlab.ProjectHook, error) {
+
```

Comment:
The code convention in the team's projects is usually no blank line at the beginning of the function. @ericjaystevens Do you have any objections on removing the blank lines?

Reply:
@ericjaystevens Do you mind applying this fix to every function introduced in the PR?